src/renderer/src/utils/style/flex.ts

```
@@ -0,0 +1,19 @@
+import type { CSSProperties } from 'react'
+import { css } from 'styled-components'
+
+interface Props {
+  flexDirection?: CSSProperties['flexDirection']
+  justifyContent?: CSSProperties['justifyContent']
+  alignItems?: CSSProperties['alignItems']
+}
+
+const flex = ({ flexDirection, justifyContent, alignItems }: Props) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)